### PR TITLE
Implements Hooks for GriefDefender & ChunkyBorder.

### PR DESCRIPTION
**Pull request by sarhatabaot**
Implements Hooks for GriefDefender & ChunkyBorder.

This fixes players being teleported into the void when using ChunkyBorder as opposed to WorldBorder.

### CODE REVIEW

File: randomteleport-plugin/pom.xml [hunk 1]
```diff
@@ -42,6 +42,12 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>${project.groupId}.pluginhook</groupId>
+            <artifactId>chunkyborder</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (Phoenix616) [inline randomteleport-plugin/pom.xml:50]: You seem to have forgotten to add the GriefDefender hook atifact here.

Comment (sarhatabaot) [reply randomteleport-plugin/pom.xml:50]: Oops, added it in.

**Comment by Phoenix616** [comment]
Sorry for not getting to this sooner. I kinda forgot about it. 🙈

**Comment by johnfriedrich** [comment]
Thanks for the merge, was about to ask for the status

**Pull request merged**